tide/src/error.rs

```diff
@@ -1,105 +1,6 @@
 use core::pin::Pin;
 use futures::future::Future;
-use http::{HttpTryFrom, Response, StatusCode};
-use http_service::Body;
 
-use crate::response::IntoResponse;
+pub use tide_core::error::*;
```

Comment:
Personal nitpick, but I don't like using glob imports outside of test modules. Would prefer all the related error types be exported explicitly.

Reply:
Updated.